config/canonical.json

```diff
@@ -34666,6 +34666,17 @@
       "shop": "watches"
     }
   },
+  "shop/wholesale|BJ's Wholesale Club": {
+    "nocount": true,
+    "tags": {
+      "brand": "BJ's Wholesale Club",
+      "brand:wikidata": "Q4835754",
+      "brand:wikipedia": "en:BJ's Wholesale Club",
```

Comment:
The `brand:wikipedia` tag needs to have underscores in it, like `"en:BJ's_Wholesale_Club"`.   
Can you change these on any of your open PRs?

Reply:
Sorry, what I wrote here was wrong. 😆